dev-infrastructure/Makefile

```diff
@@ -165,6 +165,16 @@ create-mock-identities: configurations/mock-identities.bicepparam
 		--scope "/subscriptions/$(shell az account show --query id --output tsv)" \
 		--only-show-errors
 
+# TODO: this is a workaround to enable the mock FPA to call the Check Access API with a built-in role.
+# The permissions should be reduced to match "dev-first-party-mock" role definition (ARO-17798).
+.PHONY: assign-role-for-rbac
+
+assign-role-for-rbac:
```

Comment:
Does this need to be incorporated somewhere so in case an env is recreated it is configured correctly? and if so, does it need to be idempotent?